src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -870,9 +880,9 @@ export class TerminalService implements ITerminalService {
 		return terminalIndex;
 	}
 
-	protected async _showTerminalCloseConfirmation(): Promise<boolean> {
+	protected async _showTerminalCloseConfirmation(singleTerminal?: boolean): Promise<boolean> {
```

Comment:
Reviewing on mobile and not sure why this boolean is needed

Reply:
it's needed bc the selected instances are what we care about here, not the total instance count